src/DependencyFactory/SwiftMailerFactory.php

```diff
@@ -47,8 +47,17 @@ public static function definitions()
         ];
     }
 
-    public static function buildSmtpTransport(array $relay = NULL)
-    {
+    /**
+     * @param array|null                  $relay
+     * @param \Swift_Events_EventListener ...$plugins
+     *
+     * @return \Swift_SmtpTransport
+     */
+    public static function buildSmtpTransport(
+        array $relay = NULL,
+        \Swift_Events_EventListener ...$plugins
```

Comment:
I'm not certain if this signature works if there are no plugins? Off the top of my head can't remember how it behaves following an optional argument. Could you check?

Reply:
```php
php > function test(...$var){var_dump($var);}
php > test();
array(0) {
}
```

Reply:
Awesome, LGTM then just needs adding to the changelog